### ISSUE TITLE
[Arch Linux] Black screen and cursor only after boot

### ISSUE THREAD

**Issue by RickStanley** [linuxdeepin/developer-center#366]
Unfortunately, the issue persists for me.
I tried `gsettings reset com.deepin.dde.display brightness` and got:
```
(process:9616): dconf-WARNING **: failed to commit changes to dconf: Error spawning 
command line `dbus-launch --autolaunch=aaa5bb6eaa7cd50f2af1f10000000004 --binary-syntax 
--close-stderr': Child process exited with code 1
```
; then I tried `sudo -u [my-username] dbus-launch gsettings reset com.deepin.dde.display brightness`, no errors returned, from liveusb, and also as arch-chroot. 

Additional info:
- `xf86-video-amdgpu`
- LightDM display manager

Very similar (and hopefully the same) issue: [Arch Linux] Black screen and cursor only after session login #213

**Comment by RickStanley** [comment]
I've opted to reinstall the system entirely.

**Comment by RickStanley** [comment]
Nope, the problem persists

**Comment by jouyouyun** [comment]
What's the system version?

**Comment by justforlxz** [comment]
Sorry, this issue will be closed soon. If it is necessary to discuss it again, please create a new issue.
